web/src/rendered_markdown.ts

```diff
@@ -281,6 +281,7 @@ export const update_elements = ($content: JQuery): void => {
         const $codehilite = $(this);
         const $pre = $codehilite.find("pre");
         const fenced_code_lang = $codehilite.data("code-language");
+        const $buttonContainer = $(`<div class="code-buttons-container"></div>`);
```

Comment:
Can you rewrite the logic for this container using a handlebars template? I think that'd be a lot cleaner than doing all of this with JavaScript manipulation of HTML; the template can just have a loop.

Reply:
Ohh so does that mean I need to create a new template file and add the logic for `copy_code_button` and `view_in_playground_button` using partials in a button-container?
Because I don't see a way to append the buttons in a template file using javascript.

Reply:
@shashank-23002 Yes. Study some of the existing code, including how to call partials with `render_PARTIAL_NAME()` in the JavaScript and pass in variables, and use things like `{{#if }}` conditions in [Handlebars](https://handlebarsjs.com/guide/builtin-helpers.html#if).

Reply:
Sure , I will have a look into it ASAP. Was a little busy due to semester exams.

Just wanted to ask a little doubt , if it would be better to make a new template file which would only have the code-buttons-container div?
Then we can append the buttons directly into the template file using the current javascript code.
This would be much similar to the way we append both the buttons into the <pre> tag currently.